charts/cachenator/values.yaml

```diff
@@ -1,16 +1,32 @@
 image:
   registry:
```

Comment:
This should point to a default registry/image/tag for cachenator. Whatever is latest. And we overwrite it wherever it's used internally.